packages/k4pandora/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from spack.pkg.k4.Ilcsoftpackage import Key4hepPackage, k4_add_latest_commit_as_version 
+
+
+class K4pandora(CMakePackage, Key4hepPackage):
+    """k4Pandora is a pandora app for the Key4HEP software framework"""
+
+    homepage = "https://github.com/key4hep/k4Pandora"
+    url      = "https://github.com/key4hep/k4Pandora/archive/master.tar.gz"
+    git      = "https://github.com/key4hep/k4Pandora.git"
+
+    maintainers = ['mirguest']
```

Comment:
```suggestion
    tags = ['hep', 'key4hep']
    
    maintainers = ['mirguest']
```

I usually add these tags, which let you filter spack packages and may be useful in future applications.